back/src/graphql/type/user.js

```diff
@@ -1,12 +1,5 @@
-import { conn } from '../../db/connection.js';
-import {
-  GraphQLID,
-  GraphQLInt,
-  GraphQLList,
-  GraphQLObjectType,
-  GraphQLString,
-  GraphQLBoolean,
-} from 'graphql';
+import {conn} from '../../db/connection.js';
+import {GraphQLBoolean, GraphQLID, GraphQLInt, GraphQLList, GraphQLObjectType, GraphQLString,} from 'graphql';
```

Comment:
Where is your PR checklist? I think you must never not run `yarn format` right? It is clear on the checklist. - This line > 80 characters.

Reply:
Do you just remove the checklist? I am not sure why...